R/modified_update.R

```diff
@@ -532,3 +537,98 @@ mu_calculate_rate <- function(name, dat, window, n_years) {
 
   dat
 }
+
+mu_fix_sdf7_psu <- function(meta, data){
```

Comment:
Could you add a comment explaining what needs fixing here please?  Link to an issue on youtrack if needed (though it looks like the ticket - [VIMC-1266](https://vimc.myjetbrains.com/youtrack/issue/VIMC-1266)? - has no information in it about what the problem is too).

Can this function can safely called through all modified updates in future?

Reply:
I will add a comment here. 

Good point - I have modified the code, so that this function will only run if the problem source - "SDF7 - PSU - Measles - Campaign" - is a source touchstone.